addons/point_of_sale/static/src/js/Screens/PartnerListScreen/PartnerListScreen.js

```diff
@@ -191,10 +191,10 @@ class PartnerListScreen extends PosComponent {
         let domain = [];
         const limit = 30;
         if(this.state.query) {
+            const search_fields = ["name", "parent_name", "phone_mobile_search", "email"];
```

Comment:
@khah-odoo Can you confirm that all these fields are still available in res.partner model?